libs/components/modals/src/lib/modules/modal/modal.component.ts

```diff
@@ -34,7 +37,22 @@ let skyModalUniqueIdentifier = 0;
   templateUrl: './modal.component.html',
   styleUrls: ['./modal.component.scss'],
   animations: [skyAnimationModalState],
-  providers: [SkyModalComponentAdapterService, SkyDockService],
+  providers: [
+    SkyModalComponentAdapterService,
+    SkyDockService,
+    SkyResizeObserverMediaQueryService,
+    SkyResizeObserverService,
+    {
+      provide: SkyMediaQueryService,
+      useFactory: /* istanbul ignore next */ (
```

Comment:
What is the reasoning for using `useFactory` over just `useClass`?

Reply:
Switched it. No longer applies.